lib/src/main/java/com/auth0/android/lock/LockActivity.java

```diff
@@ -372,6 +377,27 @@ public void onOAuthAuthenticationRequest(OAuthLoginEvent event) {
         webProvider.start(this, connection, extraAuthParameters, authProviderCallback, WEB_AUTH_REQUEST_CODE);
     }
 
+    private void completeDatabaseAuthenticationOnBrowser() {
+        //DBConnection checked for nullability before the API call
```

Comment:
This is called from a `callback#onFailure`. The action that registered the callback already checked for nullability, see https://github.com/auth0/Lock.Android/pull/589/files#diff-01fe7e862df1aaab8f18b1b31542c58fR404-R406.